deploy/docker/docker-compose.yml

```diff
@@ -66,15 +71,14 @@ services:
     image: ${HUB}/dolphinscheduler-api:${TAG}
     ports:
       - "12345:12345"
+    profiles: ["all"]
     env_file: .env
     healthcheck:
       test: [ "CMD", "curl", "http://localhost:12345/actuator/health" ]
```

Comment:
This should change to ```http://localhost:12345/dolphinscheduler/actuator/health``` since we have context path in ```api-server```.

Reply:
I had updated.